L1Trigger/ME0Trigger/python/me0TriggerPseudoDigis_cff.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
+## configuration to build fast L1 ME0 trigger stubs
+## pseudo pads are created from pseudo digis with 192 strips instead of 384
+## the rechits are a necessary intermediate step before the pseudo pads are used
+## as input to build pseudo stubs
+
+from SimMuon.GEMDigitizer.muonME0PseudoReDigis_cfi import *
+from RecoLocalMuon.GEMRecHit.me0RecHits_cfi import *
+from RecoLocalMuon.GEMSegment.me0Segments_cfi import *
+
+simMuonME0PseudoReDigis192 = simMuonME0PseudoReDigis.clone(
+    numberOfStrips = cms.uint32(192)
```

Comment:
This only has an effect in a single partition geometry. 
See: https://github.com/cms-sw/cmssw/blob/35ad0e0d8f48fadc1782b97e3276f1c36f5feafe/SimMuon/GEMDigitizer/python/muonME0PseudoReDigis_cfi.py#L6

I have code to make different strip multiplicities work in multi-partition geometries, and can share it if it is wanted